src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -31,11 +33,13 @@ export function QueryPageContainerComponent(): JSX.Element {
     const history = useHistory();
 
     const ISOLATE_URL: string = isolateURL;
+    const ISOLATE_COUNT_URL: string = isolateCountURL + history.location.search;

```

Comment:
consider removing upper case style because these are not constants anymore

Reply:
Done. Commit: "Change upper case style and remove "event""